src/zeit/content/cp/centerpage.py

```diff
@@ -200,10 +200,7 @@ def _fill_area_cache(self):
     _area_teasered_content = gocept.cache.property.TransactionBoundCache(
         '_v_area_teasered_content', dict)
 
-    def is_teaser_present_above(self, current_area, content):
-        return content in self._teasered_content_above(current_area)
-
-    def _teasered_content_above(self, current_area):
+    def teasered_content_above(self, current_area):
```

Comment:
Das wird in in test_automatic.HideDupesTest abgedeckt;  durch die IRenderedArea durch. Hast Du ne Idee, wie man evtl explizite Tests aufschreiben könnte, ohne für IRenderedArea Tests duplizieren zu müssen?